.github/workflows/test.yml

```
@@ -37,8 +41,22 @@ jobs:
       matrix:
         python: ['3.8']
         django: ['2.2', '3.1']
-        wagtail: ['2.11', '2.12', '2.13']
+        wagtail: ['2.11', '2.12', '2.13', '2.14']
         postgres: ['10.8']
+        include:
+          - python: '3.8'
+            django: '3.2'
+            wagtail: '2.13'
```

```suggestion
            wagtail: '2.14'
```

other than that, we can get this in